Equilobe.DailyReport.SL/ReportService.cs

```diff
@@ -159,6 +180,61 @@ private void UpdateLastSyncDate(long instanceId)
             }
         }
 
+        private void AddOrUpdateDbUsers(List<AtlassianUser> users, long instanceId)
+        {
+            using (var db = new ReportsDb())
+            {
+                var dbUsers = db.AtlassianUsers.Where(p => p.InstalledInstanceId == instanceId);
+
+                foreach (var user in users)
+                {
+                    var dbUser = dbUsers.Where(p => p.Key == user.Key).SingleOrDefault();
+
+                    if (dbUser == null)
+                        db.AtlassianUsers.Add(user);
+                    else
+                        UpdateDbUser(dbUser, user);
+                }
+
+                db.SaveChanges();
+            }
+        }
+
+        private void MarkStallingUsersAsInactive(long instanceId)
```

Comment:
I do not think we should do it this way, IsActive property actually means that the user is inactive in Jira (all rights are revoked), because Jira does not allow deleting an user. So it is more or less the same thing as deleting an user. Marking an user inactive shouldn't be an option for us. If the user doesn't have the worklogs, we should just filter him out from the dashboard. But as soon as he logs the work, he should reappear.

Reply:
Should I add a new column in the db table, IsStalling, for this thing in particular?